scipy/spatial/transform/_rotation.pyx

```diff
@@ -1417,8 +1535,98 @@ cdef class Rotation:
             return np.asarray(rotvec)
 
     @cython.embedsignature(True)
-    def as_euler(self, seq, degrees=False):
+    def _prepare_axes_compute_euler(self, seq, degrees=False, 
+                                    algorithm='from_quat'):
```

Comment:
Default values for `algorithm` and `degrees` are not necessary for this private technical method.